### ISSUE TITLE
ghcjs --interactive (GHCJSi) cannot access local CSS style sheets

### ISSUE THREAD

**Issue by hansroland** [ghcjs/ghcjs#530]
I'm using ghcjs --interactive  *(GHCJSi, version 0.2.0-7.10.3)* and would like to access a local CSS style sheet.

My page has the following header:

~~~
<head>
<title>My page"</title>
<link href="css/style.css" rel="stylesheet" type="text/css">
</head>
~~~

This translates to http://localhost:6400/css/style.css . However this CSS style sheet is never accessed. Where in the directory tree of my GHCJS project (or even of my machine) do I have to place my style sheet?

**Comment by luite** [comment]
hmm, good point. I haven't implemented this because I didn't want to expose filesystem contents automatically. perhaps a new command line option should be added to tell ghcjs the location of the static files?

**Comment by hansroland** [comment]
Yes, a command line option for the root directory for the static files would be a good idea. It would be nice, if a web page generated with the interactive version of ghcjs had the same look as the one generated non-interactively. 
Without the --interactive option, the --static-root-directory should probably be ignored

**Comment by zelinf** [comment]
Having the exactly same issue. Are there any workarounds?
